salt/metalk8s/addons/ui/config/metalk8s-ui-config.yaml

```
@@ -0,0 +1,7 @@
+#!yaml
+
+# Defaults for configuration of MetalK8s UI
+apiVersion: addons.metalk8s.scality.com
```

I know other such ConfigMaps in our repo use this format, but an apiVersion should follow the `<domain>/<version>` format:

```suggestion
apiVersion: addons.metalk8s.scality.com/v1alpha1
```